src/actions.js

```diff
@@ -1,3 +1,55 @@
 import { createAction } from 'redux-actions';
+import SC from 'soundcloud';
+import _ from 'lodash';
 
 export const addFun = createAction('ADD_FUN');
+export const getTracksFromSC = createAction('GET_TRACKS_FROM_SC');
```

Comment:
Why does this file now contain all your SC library code? `actions.js` should only contain redux action creator methods.

Reply:
Also, consider renaming this action to something like "APPEND_NEW_TRACKS". The "getting" has been taken over by the thunk action. This action (and the reducer that handles it) just do work on the state's track list and a payload of tracks.